src/redux/Books/books.js

```diff
@@ -0,0 +1,32 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// action creators
+export const addBooks = (text) => ({
+  type: ADD_BOOK,
+  payload: {
+    text,
+  },
+});
+
+export const removeBooks = (text) => ({
+  type: REMOVE_BOOK,
+  payload: {
+    text,
+  },
+});
+
+// create reducers
+const initialState = [];
+
+const bookReducer = (state = initialState, action) => {
+  const { type, payload } = action;
+  switch (type) {
+    case ADD_BOOK:
+      return state.filter((books) => books.id !== payload);
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Kindly make sure the case for `REMOVE_BOOK` is presented in your reducer. This book reducer is in charge of adding and removing books in your store.